doc/admin.md

```diff
@@ -49,4 +49,280 @@ Finally, we execute the commando package-import for the murano client library (c
+single instruction which is used to return the information of the status
+of the processes together with the queue size.
+
+    curl -v -H 'X-Auth-Token: a9a861db6276414094bc1567f664084d'
+    -X GET http://murano.lab.fiware.org:8082
+
+This operation will return the information regarding the tenant details
+of the execution of the Policy Manager
+
+    < HTTP/1.0 200 OK
+    < Date: Wed, 09 Apr 2014 08:25:17 GMT
+    < Server: WSGIServer/0.1 Python/2.6.6
+    < Content-Type: text/html; charset=utf-8
+    {
+       "versions":
+       [
+           {
+               "status": "CURRENT",
+               "id": "v1.0",
+               "links":
+               [
+                   {
+                       "href": "http://murano.lab.fiware.org:8082/v1/",
+                       "rel": "self"
+                   }
+               ]
+           }
+       ]
+    }
+
+
+For more details to use this GE, please refer to the User & Programmers Guide.
+
+### List of Running Processes
+
+Due to the Murano basically is running over the python process by using tox,
+the list of processes must be only tox process for murano-api and murano-engine:
+
+    ps -ewf | grep 'tox' | grep -v grep
+
+It should show something similar to the following:
+
+
+    UID        PID  PPID  C STIME TTY          TIME CMD
+    root       739     5  0 10:50 ?        00:00:00 /usr/bin/python /usr/local/bin/tox -e venv -- murano-api --config-file ./etc/murano/murano.conf
+    root       762   739  0 10:50 ?        00:00:02 /opt/murano/.tox/venv/bin/python .tox/venv/bin/murano-api --config-file etc/murano/murano.conf
+    root       800     5  0 10:50 ?        00:00:00 /usr/bin/python /usr/local/bin/tox -e venv -- murano-engine --config-file ./etc/murano/murano.conf
+    root       821   800  0 10:50 ?        00:00:02 /opt/murano/.tox/venv/bin/python .tox/venv/bin/murano-engine --config-file etc/murano/murano.conf
+
+In addition, mysql and rabbitmq process should be exist (or in the same server or in another one, according to configuration).
+
+
+    UID        PID  PPID  C STIME TTY          TIME CMD
+    mysql        1     0  0 10:47 ?        00:00:00 mysqld
+
+    rabbitmq     1     0  0 10:47 ?        00:00:00 /bin/sh -e /usr/lib/rabbitmq/bin/rabbitmq-server
+    rabbitmq    85     1  0 10:47 ?        00:00:00 /usr/lib/erlang/erts-7.2/bin/epmd -daemon
+    rabbitmq    98     1  0 10:47 ?        00:00:01 /usr/lib/erlang/erts-7.2/bin/beam -W w -A 64 -P 1048576 -K true -B i -- -root /usr/lib/erlang -progname erl -- -home /var/lib/rabbitmq -- -pa /usr/lib/rabbitm
+
+
+### Network interfaces Up & Open
+
+Taking into account the results of the ps commands in the previous
+section, we take the PID in order to know the information about the
+network interfaces up & open. To check the ports in use and listening,
+execute the command:
+
+    lsof -i | grep "$PID1\|$PID2"
+
+Where $PID1 and $PID2 are the PIDs of the tox processes obtained
```

Comment:
You write 2 PID, $PID1 and $PID2, but after you write 3 PIDs 762 and 821
+(redis-server) and 5604.

Apart from it, I think that it is a copy/paste from Policy Manager, rewrite it.